script/FullDeployer.s.sol

```diff
@@ -26,12 +26,8 @@ contract FullDeployer is ExtendedHubDeployer, ExtendedSpokeDeployer, AdaptersDep
     function deployFull(CommonInput memory commonInput, AdaptersInput memory adaptersInput, FullActionBatcher batcher)
         public
     {
-        if (commonInput.version == keccak256(abi.encodePacked(("3")))) _verifyAdmin(commonInput);
-
         _preDeployFull(commonInput, adaptersInput, batcher);
         _postDeployFull(batcher);
-
-        if (commonInput.version == keccak256(abi.encodePacked(("3")))) _verifyMainnetAddresses();
```

Comment:
Just moved to `run()` because theoretically, those changes only concern to real deployments, not to deployments used in EndToEnd or integration tests